service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -538,6 +539,7 @@ fun Database.Read.fetchApplicationSummaries(
                                         row.mapColumn("unit_reject_other_reason")
                                 )
                             },
+                    placementProposalStartDate = row.mapColumn("placement_plan_start_date"),
```

Comment:
I think this should be called `placementPlanStartDate` because proposal = a plan that has been sent to the unit, but this information is shown regardless of this state whenever a plan exists (and yes, the existing `placementProposalUnitName` field is also wrong).

Reply:
Fixed both